app-scroll-effects/app-scroll-effects-behavior.html

```diff
@@ -167,13 +168,13 @@
      * ```
      */
     createEffect: function(effectName, effectConfig) {
-      if (!Polymer.AppScrollEffects) {
-        throw new ReferenceError('undefined Polymer.AppScrollEffects');
+      if (!Polymer.AppLayout.scrollEffects) {
```

Comment:
Nope, because it's defined in helpers.html which is a dependency of app-scroll-effects-behavior.html. Also now I realize that we might not need to do this check anymore.